package.json

```diff
@@ -38,6 +38,7 @@
     "electron-dl": "ismaelmartinez/electron-dl",
     "electron-editor-context-menu": "1.1.1",
     "electron-native-notification": "1.2.1",
+    "electron-store": "^8.0.1",
```

Comment:
I prefer of versions are either unpinned or pinned, but not a mix

Reply:
Could you clarify?

Reply:
I think, I got it.